default.hbs

```diff
@@ -33,7 +33,7 @@
                     {{/if}}
                 </a>
                 <div class="gh-head-brand-wrapper">
-                    <button class="gh-search" data-ghost-search>{{> "icons/search"}}</button>
+                    <button aria-label="Search this site" class="gh-search" data-ghost-search>{{> "icons/search"}}</button>
```

Comment:
Would you be able to put the attribute `aria-label` after `class` by switching their position?